sse2neon.h

```diff
@@ -1692,20 +1692,16 @@ FORCE_INLINE __m128i _mm_srai_epi16(__m128i a, int imm)
 // Shifts the 4 signed or unsigned 32-bit integers in a left by count bits while
 // shifting in zeros. :
 // https://msdn.microsoft.com/en-us/library/z2k3bbtb%28v=vs.90%29.aspx
-// FORCE_INLINE __m128i _mm_slli_epi32(__m128i a, __constrange(0,255) int imm)
-#define _mm_slli_epi32(a, imm)                                   \
-    __extension__({                                              \
-        __m128i ret;                                             \
-        if ((imm) <= 0) {                                        \
-            ret = a;                                             \
-        } else if ((imm) > 31) {                                 \
-            ret = _mm_setzero_si128();                           \
-        } else {                                                 \
-            ret = vreinterpretq_m128i_s32(                       \
-                vshlq_n_s32(vreinterpretq_s32_m128i(a), (imm))); \
-        }                                                        \
-        ret;                                                     \
-    })
+FORCE_INLINE __m128i _mm_slli_epi32(__m128i a, const int imm)
```

Comment:
Remove the `const`.
After that, I think the pull request can be merged.

Reply:
@jserv I saw you previous comment that the `const` should be added.
However, based on the [_mm_slli_epi32](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_slli_epi32), it does not contain `const`.

Reply:
Yes, there is no `const` qualifier according to Intel Intrinsics. However, it is indeed an `imm8`, which should be mapped to 8-bit width.

Reply:
So should I conduct any change?